x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/WaterfallContainer/Waterfall/waterfall_helpers/waterfall_helpers.ts

```diff
@@ -72,18 +73,33 @@ interface IWaterfallItemBase {
   childIds?: Array<IWaterfallItemBase['id']>;
```

Comment:
I looks like `childIds` is no longer used. Do you agree?

Reply:
Yes, I want to remove it in another PR, because I will also remove other properties.